src/components/common/Carousels/GuideImagesCarousel.jsx

```diff
@@ -0,0 +1,32 @@
+import { Swiper, SwiperSlide } from 'swiper/react';
+import { Navigation, Pagination, Autoplay } from 'swiper/modules';
+
+import 'swiper/css';
+import 'swiper/css/navigation';
+import 'swiper/css/pagination';
+import '../../../../css/guideCarouselSwiper.css';
```

Comment:
I think `guideCarouselSwiper.css` is specific enough and cannot be reused across the project, so you can place it along with the component